modules/admin/proc-configuring-postgresql-instance-using-helm.adoc

```diff
@@ -0,0 +1,162 @@
+[id="proc-configuring-postgresql-instance-using-helm_{context}"]
+= Configuring an external PostgreSQL instance using the Helm Chart
+
+You can configure an external PostgreSQL instance by using the Helm Chart. By default, the Helm Chart creates and manages a local instance of PostgreSQL in the same namespace where you have deployed the {product-very-short} instance. However, you can change this default setting to configure an external PostgreSQL database server, for example, AWS RDS or Azure database.
+
+.Prerequisites
+
+* You are using a supported version of PostgreSQL. For more information, see the link:https://access.redhat.com/support/policy/updates/developerhub[Product life cycle page].
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+* You have installed the {product-very-short} application by using the Helm Chart.
+* Optional: You have a CA certificate, TLS private key, and TLS certificate so that you can secure your database connection by using the Transport Layer Security (TLS) protocol. For more information, refer to your PostgreSQL vendor documentation.
+
+[NOTE]
+====
+By default, {product-short} uses a database for each plugin and automatically creates it if none is found. As an administrator, know that users might need the `Create Database` privilege in addition to `PSQL Database` privileges for configuring an external PostgreSQL instance.  
+====
+ 
+
+.Procedure
+
+. Optional: Create a certificate secret to configure your PostgreSQL instance with a TLS connection: 
++
+[source,yaml]
+----
+cat <<EOF | oc -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: <crt-secret> <1>
+type: Opaque
+stringData:
+ postgres-ca.pem: |-
+  -----BEGIN CERTIFICATE-----
+  <ca-certificate-key> <2>
+ postgres-key.key: |-
+  -----BEGIN CERTIFICATE-----
+  <tls-private-key> <3>
+ postgres-crt.pem: |-    
+  -----BEGIN CERTIFICATE-----
+  <tls-certificate-key> <4>
+  # ...
+EOF  
+----
+<1> Provide the name of the certificate secret.
+<2> Provide the CA certificate key.
+<3> Provide the TLS private key.
+<4> Provide the TLS certificate key.
```

Comment:
These values are required only if some want's to use TLS with certificate **and need a key for it**.
```suggestion
<3> Optional: Provide the TLS private key.
<4> Optional: Provide the TLS certificate key.
```